parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/ParallelConsumer.java

```diff
@@ -49,6 +49,35 @@
      */
     void subscribe(Pattern pattern, ConsumerRebalanceListener callback);
 
+    /**
+     * Pause this consumer (i.e. transition to state {@link io.confluent.parallelconsumer.internal.State#paused paused} for the
+     * controller and the broker poll system).
```

Comment:
I decided to pause polling to be a bit closer to `KafkaConsumer#pause` which also suspends fetching. Let me know if you see a general issue with this approach.

Reply:
Anyway looking at your comment regarding the changes in `BrokerPoolSystem` it makes sense to leave the poller active to avoid side effects with _dead sessions_.

Reply:
Changes in `BrokerPollSystem` reverted + Javadoc reworked